Bottom: 12777dfddf9d17f32a74e6133d2451c7a5543b9c
Top:    db303d9f333cd93ee88ae7b680e325ddc00531fa
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-07-02 13:24:57 -0400

Refresh of initial-sound-driver-port

---

diff --git a/arch/arm/boot/dts/sun7i-a20-cubietruck.dts b/arch/arm/boot/dts/sun7i-a20-cubietruck.dts
index a347eff..ed40c2e 100644
--- a/arch/arm/boot/dts/sun7i-a20-cubietruck.dts
+++ b/arch/arm/boot/dts/sun7i-a20-cubietruck.dts
@@ -137,7 +137,7 @@
 			status = "okay";
 		};
 
-		codec@1c22c00 {
+		codec@01c22c00 {
 			status = "okay";
 		};
